src/shared/bus-util.c

```diff
@@ -1549,3 +1549,47 @@ int bus_message_new_method_call(
 
         return sd_bus_message_new_method_call(bus, m, locator->destination, locator->path, locator->interface, member);
 }
+
+int bus_add_implementation(sd_bus *bus, const BusObjectImplementation *impl, void *userdata) {
+        int r;
+
+        log_debug("Registering bus object implementation for path=%s iface=%s", impl->path, impl->interface);
+
+        for (const BusObjectVtablePair *p = impl->vtables; p->vtable; p++) {
+                if (!p->object_find)
```

Comment:
not a fan of this i must say. there *is* benefit of allowing object_find also for non-fallback kind of stuff. it's a different concept.

i..e object_find() can instantiate objects on demand, and that's useful even if you want to instantiate things only one or twice. i.e. think about an object that encapsulates something we store on disk. in object_find() we could load that, and return a pointer of what we just loaded. (machined does that). doing something like that makes sense independently of declaring that their are multiple objects of that same time. Hence, I think this should be kept independent of the specification of an obejct_find parameter, even if right now we usually combine both mechanisms.